src/v2/controllers/room/export-users/__tests__/export-users.ts

```diff
@@ -0,0 +1,59 @@
+import test from "ava";
+import { initializeDataSource } from "../../../../__tests__/helpers/db/test-hooks";
+import { HelperAPI } from "../../../../__tests__/helpers/api";
+import { roomRouters } from "../../routes";
+import { roomExportUsers } from "../";
+import { testService } from "../../../../__tests__/helpers/db";
+import { useTransaction } from "../../../../__tests__/helpers/db/query-runner";
+
+const namespace = "v2.controllers.room.export.users";
+
+initializeDataSource(test, namespace);
+
+test(`${namespace} - export users`, async ava => {
+    const { t, releaseRunner, commitTransaction } = await useTransaction();
+    const { createUser, createUserPhone, createRoom, createRoomJoin } = testService(t);
+
+    const roomUserCount = 10;
+    const users = [];
+    for (let i = 0; i < roomUserCount; i++) {
+        const user = await createUser.quick();
+        const userPhone = await createUserPhone.quick({
+            userName: user.userName,
+            userUUID: user.userUUID,
+        });
+        users.push({ ...user, phoneNumber: userPhone.phoneNumber });
+    }
```

Comment:
```suggestion
```suggestion
    const roomUserCount = 10;
    const users = await Promise.all(Array.from({ length: roomUserCount }, async () => {
        const user = await createUser.quick();
        const { phoneNumber } = await createUserPhone.quick({
            userName: user.userName,
            userUUID: user.userUUID,
        });

        return {
            ...user,
            phoneNumber,
        };
    }));
```
```